modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -42,7 +42,7 @@ uniform vec2 project_uViewportSize;
 uniform float project_uDevicePixelRatio;
 uniform float project_uFocalDistance;
 uniform vec3 project_uCameraPosition;
-uniform vec2 project_uCoordinateOrigin;
+uniform vec3 project_uCoordinateOrigin;
```

Comment:
Do we need change the code which set the project_uCoordinateOrigin uniform? cause it is vec3 now.

Reply:
https://github.com/uber/deck.gl/pull/3131/files#diff-c3a9d768f351000a98d9a6d74a57f38fR104